boards/msb-430/include/board.h

```diff
@@ -43,8 +43,8 @@ extern "C" {
 /**
  * @brief   Define the CPU model for the <msp430.h>
  */
-#ifndef __MSP430F1612__
-#define __MSP430F1612__
+#ifndef MSP430F1612
```

Comment:
Just realized...  Here and in the other MSP430 board: These aren't header guards. Please revert these to their original form.